cli/src/commands/jobs.rs

```diff
@@ -94,6 +94,12 @@ pub async fn handle_history(api: &mut PhylumApi, matches: &clap::ArgMatches) ->
         .and_then(|v| Filter::from_str(v).ok());
 
     if let Some(matches) = matches.subcommand_matches("project") {
+        if pretty_print {
+            print_user_warning!(
+                "`phylum history project` is deprecated, use `phylum project` instead"
+            );
+        }
+
```

Comment:
This should be moved inside the `if project_job_id.is_none()` section below since that is the case that duplicates the `phylum project` command